fuzz/src/lib.rs

```diff
@@ -27,6 +27,12 @@ pub fn generate_valid_module(
     config.canonicalize_nans = u.arbitrary()?;
     config.custom_page_sizes_enabled = u.arbitrary()?;
     config.wide_arithmetic_enabled = u.arbitrary()?;
+    config.shared_everything_threads_enabled = u.arbitrary()?;
+    if config.shared_everything_threads_enabled {
+        config.threads_enabled = true;
+        config.gc_enabled = true;
+    }
+    config.reference_types_enabled = config.reference_types_enabled || config.gc_enabled;
```

Comment:
Mind putting this in `Config::sanitize`? 

Ideally as well shared-everything-threads wouldn't require `gc_enabled` since that would help test just the shared functions/globals/etc side of things.

For the `reference_types_enabled` change here, I'm surprised that was needed due to `Config::sanitize`. Did you run into an issue where this line was required? If so we might be forgetting to sanitize something somewhere?

Reply:
> Mind putting this in Config::sanitize?

Ok, but I'm going to use the inverted form to follow what's already there (right?).

> Ideally as well shared-everything-threads wouldn't require gc_enabled since that would help test just the shared functions/globals/etc side of things.

Ok, I'll try it without that.

> For the reference_types_enabled change here, I'm surprised that was needed due to Config::sanitize. Did you run into an issue where this line was required? If so we might be forgetting to sanitize something somewhere?

This is code from months ago and I see you just added or modified `Config::sanitize` about a month ago. I think I might have just brought something that was necessary before that change. I'll try without it...

Reply:
Ah yeah it'll be inverted where shared-everything-threads will be disabled if threads is disabled (for example)

Also for reference-types aha makes sense! I think it may be the case it's no longer necessary, but would be good to double-check.